src/components/AddGrid.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div>
+      <div class="form-table"> 
+        <div>
+          <span>number of columns</span>
+        </div>       
+        <div>
+          <input type="text" name="" value=""  id="numColumms">
+        </div>
+        <div>
+          <span>number of rows</span>
+        </div>       
+        <div>
+          <input type="text" name="" value=""  id="numRows">
+        </div>
+        
+        <div>
+          <button type="button" name="button" v-on:click="addGrid">Insert Grid</button>        
+        </div>        
+      </div>
+
+      <hr>
+      <div class="form-table">
+        <div>
+          <span>number of rows</span>
+        </div>       
+        <div>
+          <input class="inputResponse" type="text" name="" value=""  id="TokenPlayer">
+        </div>
+        <div>
+          <button type="button" name="buttonJoin" v-on:click="GameJoin">Join</button>        
+        </div>
+      </div>             
+    </div>
+</template>
+
+<script>
+import EventBus from "../services/bus";
+import ServicesGame from '../services/game-create.js';
+
+export default {
+  name: "AddGrid",
+  data() {
+    return {
+      grid: "",
+      newGame : ''      
+    };
+  },
+  methods: {
+    addGrid: function() {
+      var numColumn=parseInt(document.getElementById("numColumms").value);
+      var numRow=parseInt(document.getElementById("numRows").value);
+      // alert(numColumn);
+
+      if(isNaN(numColumn) || isNaN(numRow)) {
+        alert("Ingrese rangos mayores a 0");
+        return;
+      }
+      EventBus.$emit("add-grid", numColumn, numRow);
```

Comment:
event names should be clear and represent actions, find a better one like: "size-change", etc

Reply:
Rename resolved for size-change